### ISSUE TITLE
Is it possible to release a 32bit Windows version of the installer

### ISSUE THREAD

**Issue by shalom-link** [spyder-ide/spyder#19923]
Although it's pretty uncommon, there are some systems in particular in schools, who still use Windows 10 32-bit.
Is it a major undertaking to release a 32bit version?

**Comment by dalthviz** [comment]
Hi @shalom-link thank you for the feedback! I think it could be possible although not sure what problems we could encounter will trying to build it. Just in case, the scripts we use to create the current installers are at https://github.com/spyder-ide/spyder/tree/master/installers/Windows

What do you think @spyder-ide/core-developers ? Maybe achieving something like this is more straight forward with the experimental conda-based installers @mrclary ?

**Comment by ccordoba12** [comment]
> I think it could be possible although not sure what problems we could encounter will trying to build it

Lack of wheels for our dependencies could be the main blocker for this.

> Maybe achieving something like this is more straight forward with the experimental conda-based installers

I think Conda-forge doesn't produce Windows 32bit packages anymore.

**Comment by mrclary** [comment]
> I think Conda-forge doesn't produce Windows 32bit packages anymore.

https://github.com/conda-forge/staged-recipes/issues/5640

I think they may have dropped 32bit support over 4 years ago.

**Comment by mrclary** [comment]
> Maybe achieving something like this is more straight forward with the experimental conda-based installers @mrclary ?

In principle it should be straight-forward with the conda-based installers _if_ all the architecture dependent Spyder dependencies were available. Given the above comments, I highly doubt that is the case. This means that the only option would be to build all architecture dependent Spyder dependencies. This may even have to be done on a 32bit system and I'm not sure that GitHub provides a Windows 32bit image. Probably most distressing is that Qt would have to be built...there be dragons! The oldest version of Qt compatible with Spyder will certainly not have a wheel built for Windows 32bit and I suspect that, depending on Qt's C library dependencies, the required version may be incapable of being built on a 32bit system.

Bottom line: I think we cannot support 32bit

**Comment by CAM-Gerlach** [comment]
We were just talking about Windows 32-bit support for Python itself on the Python core dev Discord, actually.

Apparently MS stopped allowing OEMs to pre-install 32-bit Wind 10 some years back. I couldn't find a great general-interest statistics source, but at least per the latest Steam survey (which is not very representative of the markets the legacy 32-bit support targets), 32-bit Windows 10 support has plunged to somewhere in the hundreds of a percent (as its not listed, as there is only 0.12% remaining for all Windows versions). Windows 7 (non-64-bit) comes in at 0.12%, but I'm not sure it even would run on that anymore, since upstream Python.org dropped support in 3,9 (though the patches to get it back are fairly small).

Of course WoW64 will remain relevant for a long time, but as Spyder is an application, not a library, and the installers are standalone, that isn't relevant for us here.
